### PR TITLE
Set feet to Ao5 for >= 2018 rounds

### PR DESCRIPTION
**Pull request by viroulep**
Some of the competitions' rounds still have mo3 as a format.

I chose not to update < 2018 competitions' rounds, which mean we would still leave some "illegal" `Round` objects in the db.

### CODE REVIEW

File: WcaOnRails/db/migrate/20180107142301_update_feet_format_in_rounds.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class UpdateFeetFormatInRounds < ActiveRecord::Migration[5.1]
+  def change
+    competitions_with_feet = Competition.includes(competition_events:[:rounds]).has_event("333ft").where("end_date >= ?", Date.new(2018, 1, 1))
+    rounds_with_feet_mo3 = competitions_with_feet.map(&:competition_events).flatten.select { |e| e.event_id == "333ft" }.map(&:rounds).flatten.select { |r| r.format_id == "m" }
+    rounds_with_feet_mo3.each do |r|
+      r.format_id = "a"
+      r.save
```
Comment (jonatanklosko) [inline WcaOnRails/db/migrate/20180107142301_update_feet_format_in_rounds.rb]: We are usually more strict in migrations, so I'd suggest `r.save!` or more concisely `r.update! format_id: "a"`.

**Comment by jonatanklosko** [comment]
Ah, make sure to run the migration and add the corresponding *structure.sql* `INSERT` statement to the diff. That's why Travis is failing =)

**Comment by viroulep** [comment]
Arf, I did run it and just "git checkout --" the structure file as there is usually a bunch of changes I don't want to commit, and forgot it did write something useful there...

I've updated the PR!

**Comment by jfly** [comment]
The code LGTM, but repeating my comment from https://github.com/thewca/worldcubeassociation.org/pull/2373#issuecomment-355831589:

> Thanks, Philippe! Is the migration necessary? I'm surprised the delegates didn't tell us about this when they added the event. Maybe it would be better to email all the delegates of affected competitions and ask them to update their event description? This would teach them to do average of 5 instead of mean of 3, just in case they didn't realize.

**Comment by viroulep** [comment]
Some of the delegates already have indicated Ao5 on their schedule, but simply didn't change it on the website.
Also visiting the event page now displays "Ao5" for feet even if the format in the db is "Mo3", as Ao5 is the default, which may be misleading.

Maybe do this *and* email the affected Delegates that their rounds formats have been changed?

**Comment by jfly** [comment]
That sounds good to me, @viroulep!

**Pull request merged**